vector/src/main/java/im/vector/app/features/settings/devices/v2/VectorSettingsDevicesFragment.kt

```diff
@@ -61,12 +114,48 @@ class VectorSettingsDevicesFragment @Inject constructor() : VectorBaseFragment<F
     }
 
     private fun initLearnMoreButtons() {
-        views.devicesListHeaderSectionOther.onLearnMoreClickListener = {
+        views.deviceListHeaderSectionOther.onLearnMoreClickListener = {
             Toast.makeText(context, "Learn more other", Toast.LENGTH_LONG).show()
         }
     }
 
     private fun cleanUpLearnMoreButtonsListeners() {
-        views.devicesListHeaderSectionOther.onLearnMoreClickListener = null
+        views.deviceListHeaderSectionOther.onLearnMoreClickListener = null
+    }
+
+    override fun invalidate() = withState(viewModel) { state ->
+        val currentDeviceInfo = state.devices()
```

Comment:
what's the difference between `state.devices()` and `state.devices`? is it possible to have a `currentDeviceInfo` and _not_ have devices in the `Success` state?